ansible/roles/nginx_vhost/tasks/main.yml

```diff
@@ -273,6 +281,16 @@
   tags:
     - nginx_vhost
 
+- name: add cors fragment if required
+  template:
+    src: "fragment_74_location_cors.j2"
+    dest: "{{nginx_conf_dir}}/vhost_fragments/{{hostname}}/http_70_location_{{ item.sort_label | default(item.path | basename) }}_74_cors"
```

Comment:
The destination is named `http_70_location`. I think that should be `http_74_location` to ensure the fragment numbering matches the template numbering for debugging purposes.

Reply:
See https similar comment below.